src/screens/Music/stacks/components/SinglesListView.tsx

```diff
@@ -0,0 +1,176 @@
+            }
+        >
+            <WrappableButtonRow>
+                <WrappableButton title={playButtonText} icon={Play} onPress={playEntity} />
+                <WrappableButton title={shuffleButtonText} icon={Shuffle} onPress={shuffleEntity} />
+            </WrappableButtonRow>
+            <View style={{ marginTop: 8 }}>
+                {trackIds.map((trackId, i) =>
+                    <TouchableHandler
+                        key={trackId}
+                        id={i}
+                        onPress={selectTrack}
+                        onLongPress={longPressTrack}
+                    >
+                        <TrackContainer
+                            isPlaying={currentTrack?.backendId === trackId || false}
+                            style={[defaultStyles.border, currentTrack?.backendId === trackId || false ? defaultStyles.activeBackground : null ]}
+                        >
+                            {
+                                tracks[trackId]?.AlbumId ?
+                                    <SingleImage source={{ uri: getImage(String(tracks[trackId]?.AlbumId)) }} style={defaultStyles.imageBackground} /> :
+                                    <SingleImage source={{ uri: getImage(String(trackId)) }} style={defaultStyles.imageBackground} /> 
+                            }
+                            
+                            <View style={[{flexDirection:'column', marginLeft: 20, marginRight:10, flexShrink: 1 }]}>
+                                <Text
+                                    style={[currentTrack?.backendId === trackId
+                                        ? { color: THEME_COLOR, fontWeight: '700' }
+                                        : defaultStyles.text,
+                                        {flexWrap: 'wrap'},
+                                    ]}
+                                    numberOfLines={2}
+                                >
+                                    {tracks[trackId]?.Name}
+                                </Text>
+                                <Text
+                                    style={[
+                                        styles.artistTrackText,
+                                        currentTrack?.backendId === trackId
+                                        ? { color: THEME_COLOR }
+                                        : defaultStyles.text
+                                    ]}
+                                >
+                                    {tracks[trackId]?.AlbumArtist}
+                                </Text>
+                            </View>
+                            <View style={{ marginLeft: 'auto'}}>
+                                <DownloadIcon trackId={trackId} />
+                            </View>
+                        </TrackContainer>
+                    </TouchableHandler>
+                )}
+                <WrappableButtonRow style={{ marginTop: 24 }}>
+                    <WrappableButton
```

Comment:
It's probably best to put this up top, as it takes a really long time to scroll down (and find it) if your library is huge.